website/common/script/content/spells.js

```diff
@@ -105,6 +105,9 @@ spells.wizard = {
     target: 'self',
     notes: t('spellWizardFrostNotes'),
     cast (user) {
+      // Check if chilling frost skill has been previously casted or not.
+      // See #12361 for more details.
+      if (user.stats.buffs.streaks === true) throw new BadRequest(t('spellAlreadyCast')(user.language));
```

Comment:
```suggestion
    cast (user, target, req) {
      // Check if chilling frost skill has been previously casted or not.
      // See #12361 for more details.
      if (user.stats.buffs.streaks === true) throw new BadRequest(t('spellAlreadyCast')(req.language));
```

@JalanshMunshi I think this is the last change needed, everything else looks good :) Basically cast accepts 3 parameters: user, target (not applicable in this case and `req`. Using req.language  is better as it allows the user to override the languages stored in their preferences.

Could you also change it for the `heal` spell at line 230? Thanks! After that I'll make a final manual testing and it should be good to go

Reply:
@paglias I am not sure I understand why is the test failing at line 30 in `test/common/ops/spells.js`. The error's `NotAuthorized` and that is what we are expecting. Can you please help?